docs/source/guide/lre-3-options.md

```diff
@@ -0,0 +1,278 @@
+---
+jupytext:
+ text_representation:
+   extension: .md
+   format_name: myst
+   format_version: 0.13
+   jupytext_version: 1.11.1
+kernelspec:
+ display_name: Python 3
+ language: python
+ name: python3
+---
+
+
+# What additional options are available when using LRE?
+
+
+In [](lre-1-intro.md), {func}`.execute_with_lre` was used to calculated the error mitigated expectation values
+In this section, we will outline the optional arguments that can be used and adjusted with this technique.
+
+
+```
+from mitiq.lre import execute_with_lre
+
+
+lre_value = execute_with_lre(
+   circuit,
+   executor,
+   degree,
+   fold_multiplier,
+   folding_method = <"noise scaling method imported from zne.scaling.folding">,
+   num_chunks = <"group a large circuit into a circuit with fewer layers">
+)
+```
```

Comment:
We don't want this cell to be executed during the docs build.

Reply:
I had the impression that the suggestion above only highlights the code, but does not make it executable (we aren't using the `{code-cell}` tag). Did you find otherwise when trying it?